app/_src/reference/kubernetes-annotations.md

```diff
@@ -605,6 +605,14 @@ spec:
         [...]
 ```
 
+{% if_version gte:2.4.x %}
+### `kuma.io/wait-for-dataplane-ready`
+
+Define if you want the kuma-sidecar container to wait for the dataplane to be ready before exiting.
```

Comment:
`to be ready before exiting.` not sure if that makes sense. Maybe ``to be ready before starting application container